tests/experiment/data/results_for_testing_parameters.csv

```diff
@@ -0,0 +1,212 @@
+#Procedure: <test_results.test_parameter_reading.<locals>.DummyProcedure>
+#Parameters:
+#	Directory string: //test directory with space/test_filename.csv
```

Comment:
It's probably not material to this test, but this path starting with `//` looks really strange. :-)

Reply:
Yeah, you're right, it's a bit strange but not material; I'll change it anyway.